.github/workflows/ci.yaml

```diff
@@ -87,6 +87,17 @@ jobs:
           name: ruff
           path: target/debug/ruff
 
+  cargo-fuzz:
+    runs-on: ubuntu-latest
+    name: "cargo fuzz"
+    steps:
+      - uses: actions/checkout@v3
+      - name: "Install Rust toolchain"
+        run: rustup show
+      - uses: Swatinem/rust-cache@v2
+      - run: cargo install cargo-fuzz
+      - run: cargo fuzz build -s none
```

Comment:
Improvement (doesn't have to be part of this PR and we face the same problem with benchmarks). It would be great if we could run the fuzzer selectively 

* parser tests: Only when the `Cargo.toml` changes (as an over approximation for RustPython changed)
* linter: changes to any non-formatter or cli related crate?

Reply:
Definitely would be cool to run the fuzzers for a short amount of time! To do so, we'd need the following in CI:
 - caching of the fuzzer corpus
 - 10 minute fuzz runs

That said, Ruff isn't quite resistant enough to fuzzing yet to justify adding it to the CI. Probably all your builds would "break" due to the fuzzer finding things that simply haven't been fixed yet. I haven't even properly confirmed that I've found "all" the potential issues yet.